integration/create_test.go

```diff
@@ -26,6 +29,94 @@ import (
 	"github.com/FerretDB/FerretDB/integration/setup"
 )
 
+func TestCreateStress(t *testing.T) {
+	setup.SkipForPostgresWithReason(t, "https://github.com/FerretDB/FerretDB/issues/1206")
+
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t) // no providers there, we will create collections concurrently
+	db := collection.Database()
+
+	collNum := runtime.NumCPU() * 10
```

Comment:
That should be `runtime.GOMAXPROCS(-1)` as we are interested in the number of CPU cores ("procs") that the Go scheduler is allowed to use, not just in the number of CPU cores

Reply:
Doh, too late!